platform-sdk/swirlds-virtualmap/src/test/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCacheTest.java

```diff
@@ -376,25 +368,18 @@ void buildATree() {
         // At this point, we have built the tree successfully. Verify one more time that each version of
         // the cache still sees things the same way it did at the time the copy was made.
         final VirtualNodeCache<TestKey, TestValue> cache4 = cache;
-        validateTree(
```

Comment:
Why validateTree check is being updated even though it was not updated here --> https://github.com/hashgraph/hedera-services/pull/15575/files#diff-3afb593ce7e9ff6aa9114281036ec2fe0f28295471699388e95e92c329fde932 ? 🤔

Reply:
Good question. The fix for 15448 broke a few tests, so there was a follow-up fix, see https://github.com/hashgraph/hedera-services/issues/16911 for details. This fix is reverted, too